src/main/java/org/minions/devfund/mauricioz/WhackAMoleApplication.java

```diff
@@ -0,0 +1,46 @@
+package org.minions.devfund.mauricioz;
+
+import java.util.Scanner;
+
+/***
+ * WhackAMoleApplication class to start Whackamole game.
+ */
+public class WhackAMoleApplication {
+
+    /**
+     * constructor.
+     */
+    protected WhackAMoleApplication() { }
+
+    /**
+     *main method to manage Whackamole game.
+     * @param args default value
+     */
+    public static void main(final String[] args) {
+        final int attemptsNumber = 50;
+        final int boardDimension = 10;
+        int positionX;
+        int positionY;
+        Scanner scanner = new Scanner(System.in, "UTF-8");
+        WhackAMole game = new WhackAMole(attemptsNumber, boardDimension);
+        System.out.println("Game is Started.........");
+        game.printGridToUSer();
+        game.printBoard();
+        while (game.getAttemptsLeft() > 0) {
+            System.out.println("Enter a value for X coordinate (0-9): ");
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Replace this use of System.out or System.err by a logger. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS106)